storage/merkle_path_test.go

```diff
@@ -0,0 +1,93 @@
+package storage
+
+import (
+	"testing"
+)
+
+type bitLenTestData struct {
+	input    int64
+	expected int
+}
+
+type calcPathTestData struct {
+	treeSize     int64
+	leafIndex    int64
+	expectedPath []NodeID
+}
+
+// Expected paths built by examination of the example 7 leaf tree in RFC 6962. When comparing
+// with the document remember that our storage node layers are always populated from the bottom up.
```

Comment:
The fact that our storage works bottom-up shouldn't impact the proofs that are generated though, right?